apps/web/src/hooks/useSetPrimaryBasename.ts

```diff
@@ -22,13 +22,18 @@ import useWriteContractWithReceipt from 'apps/web/src/hooks/useWriteContractWith
 */
 
 type UseSetPrimaryBasenameProps = {
-  secondaryName: BaseName;
+  basename: BaseName;
 };
 
-export default function useSetPrimaryBasename({ secondaryName }: UseSetPrimaryBasenameProps) {
+export default function useSetPrimaryBasename({ basename }: UseSetPrimaryBasenameProps) {
   const { address } = useAccount();
   const { logError } = useErrors();
 
+  const { currentWalletIsProfileOwner } = useUsernameProfile();
+  const { basenameChain: secondaryBaseChain } = useBasenameChain(basename);
```

Comment:
nit: this name sounds like it's a secondary chain, not the name associated with the secondary name

nit2: feels a little weird to remove most "secondary" references, but add this one

Reply:
renamed